src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -484,3 +485,26 @@ def get_asic_presence_list():
                     # asic is asid id: asic0, asic1.... asicN. Get the numeric value.
                     asics_list.append(int(get_asic_id_from_name(asic)))
     return asics_list
+
+def get_docker0_ip():
+    """
+    @summary: This function will return docker0 ipv4 and ipv6 addresses
+              for multi_asic platform.
+              This function will return None for single asic platform.
+    @return: tuple containing docker0 ipv4 and ipv6 address
+    """
+    docker0_v4 = None
+    docker0_v6 = None
+
+    # return (None, None) for single asic platform
+    if not is_multi_asic():
```

Comment:
>is_multi_asic [](http://example.com/codeflow?start=11&length=13)

Is it harmful to let single-asic return the proper ips?

Reply:
This helper function is added so that we could use this to get docker0 ip for multi-asic platform.
Most likely we will need docker0 information only for multi-asic platform.
For example, we want to update SNMP_AGENT_ADDRESS_CONFIG table in config_db with docker0 and mgmt IP in miniraph parser and we could avoid adding the single/multi-asic in minigraph parser if we ensure that required IP is passed from this helped function.